components/Homepage/Featured.tsx

```diff
@@ -1,13 +1,165 @@
+        <h2 className='flex gap-3'>
+          Featured
+          <div className='relative w-fit'>
+            <div className='relative z-10'>Projects</div>
+            <div className='absolute bottom-0 z-0 h-5 w-full bg-accent' />
+          </div>
+        </h2>
+      </section>
+      <section className='mt-10 flex h-full flex-col justify-between gap-10'>
+        <section className='padding-layout relative w-full'>
+          <section className='relative flex h-[440px] w-full justify-around rounded-[20px] bg-blue-500'>
+            <section className='flex w-1/2 flex-col justify-center gap-8  pl-20'>
+              <h2 className='text-5xl font-bold text-white-900'>
+                Morrent - A Car <br /> Rental Application
+              </h2>
+              <section className='flex w-full gap-5'>
+                <button className='rounded-md bg-white-900/20 p-3 text-sm font-normal text-white-900 transition-all hover:scale-105'>
+                  ReactJS, Next.js
+                </button>
+                <button className='rounded-md bg-white-900/20 p-3 text-sm font-normal text-white-900 transition-all hover:scale-105'>
+                  Supabase, Nodejs
+                </button>
+              </section>
+              <Link href='/'>
+                <p className='flex flex-row items-center gap-2.5 text-lg font-semibold text-white-900'>
+                  See Project Details
+                  <Image
+                    src='./icons/arrowlarge.svg'
+                    width={35}
+                    height={0}
+                    alt='arrowlarge'
+                  />
+                </p>
+              </Link>
+            </section>
+            <section className='relative flex h-full w-1/2 justify-center '>
+              <Image
+                src='./img/laptop.svg'
+                alt='laptop'
+                width={564}
+                height={330}
+                className='relative'
+              />
+              <Image
+                src='./img/morrent.svg'
+                alt='laptop'
+                width={460}
+                height={0}
+                className='absolute left-[125px] top-20 rounded-[8px]'
+              />
+            </section>
+          </section>
+        </section>
+        <section className='padding-layout relative w-full'>
+          <section className='relative flex h-[440px] w-full justify-around rounded-[20px] bg-emerald-500'>
+            <section className='relative flex h-full w-1/2 justify-center '>
+              <Image
+                src='./img/laptop.svg'
+                alt='laptop'
+                width={564}
+                height={330}
+                className='relative'
+              />
+              <Image
+                src='./img/jobit.svg'
+                alt='laptop'
+                width={460}
+                height={0}
+                className='absolute left-[125px] top-20 rounded-[8px]'
+              />
+            </section>
```

Comment:
I see this section being repeated twice. Create a reusable component instead of copy pasting.